### ISSUE TITLE
fail to "npm install gdal"

### ISSUE THREAD

**Issue by wxzgis** [naturalatlas/node-gdal#272]
make: *** [Release/obj.target/gdal/src/utils/fast_buffer.o] Error 1
make: Leaving directory `/usr/local/wxz/prj/test/node_modules/gdal/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/wxz/node/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:223:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 3.10.0-514.26.2.el7.x86_64
gyp ERR! command "/usr/local/wxz/node/bin/node" "/usr/local/wxz/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64/gdal.node" "--module_name=gdal" "--module_path=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64" "--napi_version=5" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v72"
gyp ERR! cwd /usr/local/wxz/prj/test/node_modules/gdal
gyp ERR! node -v v12.15.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/wxz/node/bin/node /usr/local/wxz/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64/gdal.node --module_name=gdal --module_path=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64 --napi_version=5 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v72' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/usr/local/wxz/prj/test/node_modules/gdal/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:223:5)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
node-pre-gyp ERR! System Linux 3.10.0-514.26.2.el7.x86_64
node-pre-gyp ERR! command "/usr/local/wxz/node/bin/node" "/usr/local/wxz/prj/test/node_modules/gdal/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /usr/local/wxz/prj/test/node_modules/gdal
node-pre-gyp ERR! node -v v12.15.0
node-pre-gyp ERR! node-pre-gyp -v v0.13.0
node-pre-gyp ERR! not ok 
Failed to execute '/usr/local/wxz/node/bin/node /usr/local/wxz/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64/gdal.node --module_name=gdal --module_path=/usr/local/wxz/prj/test/node_modules/gdal/lib/binding/node-v72-linux-x64 --napi_version=5 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v72' (1)
npm WARN test@1.0.0 No description
npm WARN test@1.0.0 No repository field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gdal@0.9.9 install: `node-pre-gyp install --fallback-to-build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gdal@0.9.9 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-04-05T16_18_56_389Z-debug.log

**Comment by contra** [comment]
Not trying to plug my own thing but I forked this project and it is working on all node versions - https://github.com/contra/node-gdal-next

**Comment by brianreavis** [comment]
Fixed by https://github.com/naturalatlas/node-gdal/releases/tag/v0.10.0

P.S. Thanks for stepping in @contra; no hard feelings – really appreciate it! I mentioned your fork in the readme for those interested: https://github.com/naturalatlas/node-gdal#alternate-fork. I need to dig into your fork to see if there's a path to re-merging. The biggest challenge for us (internally) is dealing with that major PROJ change that you've noted. For that reason, I just updated node-gdal to GDAL 2.4.4 (instead of going all the way to 3.x).
